core/client/fs/src/main/java/alluxio/client/file/cache/LocalCacheManager.java

```diff
@@ -722,6 +722,9 @@ private static void registerGauges(long cacheSize, MetaStore metaStore) {
       MetricsSystem.registerGaugeIfAbsent(
           MetricsSystem.getMetricName(MetricKey.CLIENT_CACHE_SPACE_USED.getName()),
           metaStore::bytes);
+      MetricsSystem.registerGaugeIfAbsent(
+          MetricsSystem.getMetricName(MetricKey.CLIENT_CACHE_PAGE_COUNT.getName()),
```

Comment:
I think there has been a metrics key called client_cache_pages.    https://github.com/Alluxio/alluxio/blob/a10823a9523e200f1665228cd4eb3ea7659a0d15/core/common/src/main/java/alluxio/metrics/MetricKey.java#L1488

Reply:
If you just wanna count the total number of the pages in our local cache, why not just use this metrics?

https://github.com/Alluxio/alluxio/blob/f2825833fbf5d473fe7265aae749e8b3d52df143/core/client/fs/src/main/java/alluxio/client/file/cache/DefaultMetaStore.java#L146

https://github.com/Alluxio/alluxio/blob/f2825833fbf5d473fe7265aae749e8b3d52df143/core/client/fs/src/main/java/alluxio/client/file/cache/DefaultMetaStore.java#L71